README.md

```diff
@@ -7,35 +7,40 @@
 Describe the differences between a SQL and NoSQL database, and when you might use each.
 
 ```text
-Your answer...
+Sql databases are relational. The tables relate to each other via foreign keys. NoSQL are non-relational and don't require a schema, however, one can embed data in order to create relationships.
 ```
 
 ### Question #2
 
 What's wrong with this Mongoose code and how might we fix it?
 
+
 ```js
-var results = AuthorModel.find({name: "Bob"});
-console.log(results);
+var results = AuthorModel.findOne({name: "Bob"})
+  console.log(results);
+
+
 ```
 
 > Hint: Assuming there is a document with a name of "Bob", why does `results` not contain an author model on the second line?
 
 ```js
-// Your answer...
+.find returns an array. to get the specific object use either findOne() or results[0].
```

Comment:
We were looking for a code response here, correcting the broken snippet above through use of a callback:
```js
AuthorModel.find({name: "Bob"}, (results)=>{
    console.log(results);    
});
```